frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -841,7 +842,17 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
       }
       return tableItem
     })
-    return { tableItems, totalCount: (isPreProcessed && resultCounts?.itemCount) || tableItems.length }
+    // allTableItems is the complete set of table items used for export
+    const allTableItems: ITableItem<T>[] = items.map((item) => {
```

Comment:
From Kevin: this logic is being used twice, once for each set of tableItems. If we setup the tableItem array first then apply filters, we could avoid this redundancy.